lib/encryptor.rb

```diff
@@ -0,0 +1,50 @@
+require_relative './cryptor'
+require_relative './key'
+require_relative './offset'
+
+class Encryptor < Cryptor
+
+  attr_reader :encrypt_key, :encrypt_offset, :encrypt_shift
+
+  def initialize(message, key = '', date = '') 
+    super(message, key, date) 
```

Comment:
the `initialize` method of the `Encryptor` class inherits the `initialize` method of the Cryptor class